api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java

```diff
@@ -74,4 +87,61 @@ public Person getPersonByUuid(String uuid) {
 		        .createAlias("attributeType", "pat").add(eq("pat.uuid", personAttributeTypeUuid)).add(eq("voided", false))
 		        .list();
 	}
+	
+	@Override
+	public Collection<Person> searchForPeople(StringOrListParam name, TokenOrListParam identifier, TokenOrListParam gender,
```

Comment:
@CaptainDaVinci I would suggest you to write test for this as I am facing issues while implementing this for `Encounters` Don't know but my tests are failing

Reply:
Thanks! I'll update the tests in sometime.